.github/workflows/beta.yml

```diff
@@ -4,27 +4,50 @@ on:
   push:
     branches:
       - 'beta'
+  workflow_dispatch: 
 
+env:
+  IMAGE: toddrob/searcharr
 jobs:
   docker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+      - 
+        uses: actions/checkout@v2
       -
         name: Login to DockerHub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      -
+      - if: startsWith(github.ref, 'refs/heads/master')
+        run: echo "TAG=latest" >> $GITHUB_ENV
+      - if: startsWith(github.ref, 'refs/tags')
+        run: echo "TAG=$(git describe --tags)" >> $GITHUB_ENV
+      - name: upgrade apt packages..
+        run: sudo apt-get update -y && sudo apt-get upgrade -y
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - 
         name: Build and push
         uses: docker/build-push-action@v3
         with:
-          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          context: ./
+          file: ./Dockerfile
           push: true
-          tags: toddrob/searcharr:beta
+          platforms: linux/amd64,linux/arm64,linux/arm/v7,linux/arm64/v8
+          tags: toddrob/searcharr:latest
+  
+      - name: Docker Hub Description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+         username: ${{ secrets.DOCKERHUB_USERNAME }}
+         password: ${{ secrets.DOCKERHUB_TOKEN }}
+         repository: toddrob/searcharr
+         short-description: ${{ github.event.repository.description }}
```

Comment:
Please remove. I would prefer to keep the description on docker hub as it is currently, pointing back to github. Especially because this would update the docker repository even when a change is only in beta.

Reply:
aight 👍

Reply:
@toddrob99 pushed it accordingly